<launch>
  <!-- Input -->
  <arg name="input/tracks" default="~/input/tracks" />
  <!-- Output -->
  <arg name="output/noise_tracks" default="~/output/noise_tracks" />
  <arg name="output/filtered_tracks" default="~/output/filtered_tracks" />
  <!-- Parameter -->
  <arg name="velocity_angle_threshold" default="1.0472" />

  <!-- Node -->
  <node pkg="radar_tracks_noise_filter" exec="radar_tracks_noise_filter_node" name="radar_tracks_noise_filter" output="screen">
    <!-- Input -->
    <remap from="~/input/tracks" to="$(var input/tracks)"/>
    <!-- Output -->
    <remap from="~/output/noise_tracks" to="$(var output/noise_tracks)"/>
    <remap from="~/output/filtered_tracks" to="$(var output/filtered_tracks)"/>
    <!-- Parameter -->
    <param name="velocity_angle_threshold" value="$(var velocity_angle_threshold)" />
  </node>
</launch>
